velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,25 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct WeibullCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double a, double b, double value) {
+    constexpr double kInf = std::numeric_limits<double>::infinity();
```

Comment:
Make this a static variable. static constexpr double kInf = ....

Reply:
Fixed.